xmr-btc/src/config.rs

```diff
@@ -39,20 +41,22 @@ mod mainnet {
 
     pub static BITCOIN_AVG_BLOCK_TIME: Lazy<Duration> = Lazy::new(|| Duration::from_secs(10 * 60));
 
-    pub static MONERO_MAX_FINALITY_TIME: Lazy<Duration> =
-        Lazy::new(|| Duration::from_secs_f64(15f64 * 1.5 * 2f64 * 60f64));
+    pub static MONERO_FINALITY_CONFIRMATIONS: u32 = 10;
+
+    pub static MONERO_AVG_BLOCK_TIME: Lazy<Duration> = Lazy::new(|| Duration::from_secs(2 * 60));
```

Comment:
```suggestion
    pub static MONERO_AVG_BLOCK_TIME: Lazy<Duration> = Lazy::new(|| 2.minutes());
```
nitpick, there is a trait for that

Reply:
Leaving this for now as it required inclusion go time crate